source/uwp/Renderer/lib/InputValue.cpp

```diff
@@ -184,13 +184,13 @@ namespace winrt::AdaptiveCards::Rendering::Uwp
                 if (max)
                 {
                     double maxDouble = max.Value();
-                    isValid = (currentDouble <= maxDouble);
+                    isValid &= (currentDouble <= maxDouble);
                 }
 
                 if (min)
                 {
                     double minDouble = min.Value();
-                    isValid = (currentDouble >= minDouble);
+                    isValid &= (currentDouble >= minDouble);
```

Comment:
I don't think this change changes anything :)

Reply:
I added this because we reset the value at ln. 193

So say we failed the max condition (187), but we passed the min condition (193), we were returning `true` when we should return `false || true = false`

Reply:
I see :)